libbpf-tools/Makefile

```diff
@@ -49,6 +49,7 @@ APPS = \
 	solisten \
 	statsnoop \
 	syscount \
+	sysinjector \
```

Comment:
>BCC has a tool called inject, should this be renamed to inject ?

No. _sysinjector_ is not based on the BCC's inject. Actually, this tool modifies the return code of syscall instead of kmalloc, bio, alloc_page.
The objective of this tool shows how to implement chaos engineering tools for the application layer using libbpf .

Reply:
Same functionality, no ?

Reply:
>Same functionality, no ?
No.

Reply:
I won't insist on this, but I think we can extend this tool to support the inject functionalities and replace it.

Reply:
I'm sorry for the late reply.
There is a difference in concept between _inject_ and _sysinjector_.
So I think _sysinjector_ should be independent from _inject_.
However, I also want to implement _inject_  using libbpf in the near future.